lib/taskana-core/src/main/java/pro/taskana/task/api/models/Task.java

```diff
@@ -65,100 +67,106 @@
   void setDescription(String description);
 
   /**
-   * Sets the Classification key that - together with the Domain from this Task's work basket -
-   * selects the appropriate {@link Classification} for this Task.
+   * Sets the {@linkplain Classification#getKey() key} of the {@linkplain Classification} that -
+   * together with the {@linkplain WorkbasketSummary#getDomain() domain} from the {@linkplain
+   * pro.taskana.workbasket.api.models.Workbasket Workbasket} of the Task - selects the appropriate
+   * {@linkplain Classification} for this Task.
    *
-   * @param classificationKey the classification key for the Task
+   * @param classificationKey the {@linkplain Classification#getKey() key} of the {@linkplain
+   *     Classification} for the Task
    */
   void setClassificationKey(String classificationKey);
 
   /**
-   * Returns the key of the Workbasket where the Task is stored in.
+   * Returns the {@linkplain WorkbasketSummary#getKey() key} of the {@linkplain
+   * pro.taskana.workbasket.api.models.Workbasket Workbasket} where the Task is stored in.
    *
    * @return workbasketKey
    */
   String getWorkbasketKey();
 
   /**
-   * Returns a Map of custom Attributes.
+   * Returns a Map of customAttributes.
```

Comment:
"customAttributes" or "custom attributes"?

Reply:
The reasoning behind "customAttributes" is that "customAttributes" is an attribute name.